File: 01_basics/01_03_count_numbers/CountNumbers.java
public class CountNumbers {
    public static void main(String[] args) {
        int num = 12345;

        System.out.println("Number of digits in " + num + " is: " + countDigits(num));
    }
    private static int countDigits(int n) {
        int x = n;
        int count = 0;
        
        while (x != 0) {
            x = x / 10;
            count++;
        }
        return count;
    }
}

File: 01_basics/01_03_count_numbers/count_numbers.js
const countDigits = (n) => {
    let x = n;
    let count = 0;

    while (x != 0) {
        x = Math.floor(x / 10);
        count++;
    }

    return count;
}

const num = 12345;
console.log("Number of digits in " + num + " is: " + countDigits(num));